docker-compose.yml

```diff
@@ -36,7 +36,7 @@ services:
       retries: 15
 
   redis:
-    image: redis:7.1 # match aws elasticache
+    image: redis:7 # we use 7.1 on aws, but redis pulled that tag....
```

Comment:
😬 good catch I was wondering why I was having trouble building

Reply:
so... annoying.. why would you remove a published tag!!!